mypy.ini

```diff
@@ -0,0 +1,1070 @@
+[mypy]
+python_version = 3.9
```

Comment:
The next Astropy release is 3.10. Might be worth making the switch here now.

Reply:
I've been looking forward to updating PlasmaPy to 3.10+ because of the typing improvements for the last 38 months! 😹